plugins/ws.js

```diff
@@ -109,7 +109,7 @@ module.exports = function (opts) {
       else
         port = opts.port
 
-      var host = (scope == 'public' && opts.external) || scopes.host(scope)
+      var host = opts.host || (scope == 'public' && opts.external) || scopes.host(scope)
```

Comment:
Shouldn't this also be public, then host, then scopes?

Reply:
Yep, great catch.